ivy/functional/ivy/sorting.py

```diff
@@ -254,11 +252,10 @@ def sort(
 def searchsorted(
     x: Union[ivy.Array, ivy.NativeArray],
     v: Union[ivy.Array, ivy.NativeArray],
-    /,
+    side: str = "left",
```

Comment:
Why exactly are we moving these to be positional only? we should follow the Array API standard, but given that they don't have `searchsorted` in their standard, we follow the one that is most similar, in our case `argsort` and `sort`, therefore we don't need to do these changes.